### ISSUE TITLE
Adruino Pro Compiler Error

### ISSUE THREAD

**Issue by d0natz** [F-Army/arduino-dw1000-ng#187]
![DW1000ng](https://user-images.githubusercontent.com/69391145/99143394-4e95e000-26a0-11eb-9cd0-c9f0b65b1e41.jpg)

**Comment by Sonic0** [comment]
Hi @d0natz, details, screenshots, or anything else?

**Comment by d0natz** [comment]
Hi Sonic...

My apologies...
I was not aware that I have pressed the submit button.
I was able to solve the problem by just re-downloading your DW1000-ng library.
Please disregard this issue.
I will try to delete this issue now...

Regards,
Ω

**Comment by d0natz** [comment]
Hi Sonic,

It seems that I cannot delete the entry.
Thus, I just uploaded the image of the compiler error -- maybe some people may encounter it in the future.

★SOLUTION★
①DELETE "DW1000-ng" in Arduino IDE library
②DOWNLOAD a fresh "DW1000-ng" library
③ADD a freshly downloaded "DW1000-ng" to Arduino IDE library

Regards,
Ω

**Comment by Bartvelp** [comment]
Hey that's my code, not sure why a redownload would compile.
I just saw that the default arduino EEPROM class does not implement `EEPROM.begin`, only the ESP8266 arduino core.
Should the EEPROM.begin be in an `#ifdef` @Sonic0 ?

**Comment by d0natz** [comment]
Hi @Bartvelp,

Just  a brief history...

**2020Oct 30** 
First time I downloaded the "DW1000-ng" library. 
Compilation for ESP8266 & ESP32 was successful, but not for Arduino-ProMini.

**2020Nov14**
I tried to re-download the "DW1000-ng" library, and compilation for Arduino-ProMini has been successful.
I am not sure though, if there were some minor updates with "DW1000-ng" library from Oct 30 ~ Nov 13.

Regards,
Ω

**Comment by Sonic0** [comment]
Yes @Bartvelp , we didn't make any test before merge and I had to push a quick fix (look at the master code). Probably @d0natz had the previous version of the code (yours). 
On my NodeMCU 1.0 I have been problem with your code and I made code compitible only with Arduino. Sorry @Bartvelp , I should have talk about it with you, but I have no to much time to spend on this project.

**Comment by Sonic0** [comment]
Now, @Bartvelp we could try to re-insert compatility with ESP8266 (and for ESP32?) if you would like to spend time on this feature.

**Comment by Bartvelp** [comment]
Ah I see the latest commits. No problem obviously, glad it could be fixed quickly.
